Makefile

```diff
@@ -169,7 +169,7 @@ fix: install
 
 # TESTS ########################################################################
 
-PYTEST := $(BIN_)py.test
+PYTEST := FLASK_ENV=test $(BIN_)py.test
```

Comment:
We have have been doing this all along. No wonder test data was appearing in my database...